Dockerfile

```
@@ -1,14 +1,12 @@
-FROM node:20-alpine3.17
+FROM node:23.10.0-alpine3.21
 
-RUN apk add --no-cache python3 make g++  \
-    && npm install -g pnpm
+RUN apk add --no-cache python3 make g++ \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)